bitnami/kube-prometheus/templates/exporters/core-dns/servicemonitor.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.coreDns.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "kube-prometheus.fullname" . }}-coredns
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: {{ template "kube-prometheus.fullname" . }}-coredns
+spec:
+  jobLabel: component
```

Comment:
In this case the jobLabel should be `k8s-app` as it is the label which will give the Prometheus Target job a name, please check on [here](https://github.com/bitnami/charts/pull/3727/files?file-filters%5B%5D=.yaml#diff-4d6eb5b8818cb65a19f520d0ce982638R20).

Reply:
As coredns labels its pods with the following labels:
```yaml
Name:                 coredns-6955765f44-qxpbp
Namespace:            kube-system
Labels:               k8s-app=kube-dns
                      pod-template-hash=6955765f44
```

Therefore in this case, it should be used the `jobLabel: k8s-app`. Check the comment regarding kube-controller-manager for more information.

Reply:
yes, right - corrected